src/test/lib/logical_query_plan/except_node_test.cpp

```diff
@@ -23,17 +24,43 @@ class ExceptNodeTest : public BaseTest {
     _except_node = ExceptNode::make(SetOperationMode::Positions);
     _except_node->set_left_input(_mock_node1);
     _except_node->set_right_input(_mock_node1);
+
+    _except_node_without_right_input = ExceptNode::make(SetOperationMode::Positions);
+    _except_node_without_right_input->set_left_input(_mock_node1);
   }
 
   std::shared_ptr<MockNode> _mock_node1, _mock_node2, _mock_node3;
-  std::shared_ptr<ExceptNode> _except_node;
+  std::shared_ptr<ExceptNode> _except_node, _except_node_without_right_input;
```

Comment:
Should it be:
```suggestion
  std::shared_ptr<ExceptNode> _except_node;
  std::shared_ptr<ExceptNode> _except_node_without_right_input;
```?
I just chose to continue with this style because it was already heavily used in the surrounding lines (e.g. see _mock_nde1, ...) .

Reply:
Exactly. We introduced this guideline recently but did not check every test file for it. So if you encounter stuff that should be changed, please change it.